ddl/ddl.go

```diff
@@ -347,6 +353,9 @@ func (d *ddl) Start(ctxPool *pools.ResourcePool) error {
 		}
 
 		go d.schemaSyncer.StartCleanWork()
+		if config.TableLockEnabled() {
```

Comment:
Do we need to add `d.wg.Add(1)`?

Reply:
Already check `d.ctx.Done` in the `startCleanDeadTableLock`, so no need `wg.Add(1)`?

Reply:
If we don't add `wg.Add(1)`, maybe the leak test will fail?

Reply:
great, done.